win32/tests/threadpool_win32_int/threadpool_win32_int.c

```diff
@@ -1058,6 +1191,69 @@ TEST_FUNCTION(chaos_knight_test)
     execution_engine_dec_ref(execution_engine);
 }
 
+// test used for detect race condition between timer_restart/timer_cancel and timer destory, failed due to the race condition for the current code, will uncomment after the fix
+//TEST_FUNCTION(chaos_knight_test_with_timers_no_lock)
```

Comment:
>chaos_knight_test_with_timers_no_lock [](http://example.com/codeflow?start=16&length=37)

Isn't this chaos test stronger than the other one? Because it has no locks supposedly ...?

If there is no other real difference between the 2 tests, we should remove the existing test (seems to do quite the same)

Reply:
removed the existing test